lib/learn_test/dependencies/phantomjs.rb

```diff
@@ -33,6 +31,12 @@ def phantom_installed_on_mac?
       def phantom_installed_on_linux?
         phantom_installed?
       end
+      def self.check_installation
+        new.check_installation
+      end
+
+      def check_installation
```

Comment:
i think i was refactoring some logic out of that somewhere else and didn't clean it up, rming